packages/amplication-prisma-db/project.json

```diff
@@ -78,7 +78,7 @@
     "db:clean": {
       "executor": "nx:run-commands",
       "options": {
-        "command": "ts-node scripts/clean.ts",
```

Comment:
Didn't we move away from a custom clean script to a native prisma command?

Reply:
That is our current prisma-db (not generated by amplication) and it was pointing to a missing file :S

Reply:
Sure, it makes sense. Fixed across the services